focuspoints.lrdevplugin/focus_points/pentax/pentax k-50.txt

```diff
@@ -2,8 +2,6 @@
 -- Submitted by: roguephysicist
 -- 16.3 MP, 4928x3264
 -- AF System: 11 points, 9 cross-type
--- selected points: AFPointSelected
```

Comment:
Using the center point has some serious advantages, though. Think about any camera with a grid of focus points, like most Nikons, Pentax, etc. You immediately know that your center point will be centered at `(0.5*width,0.5*height)`. It's also very likely that the row and column containing the center point will also share a coordinate with the center point. This is very intuitive and easy to visualize.

Secondly, the "size" of the focus points in somewhat vague. I mean, what is it? The size of the sensor? The spot on the viewfinder? In our case, it's something that's being set in the camera file, and can be freely adjusted to more accurately represent the camera, or perhaps for visual effect. If the points are top-left aligned, they must be aligned for exactly one specific size. Any adjustment to the dimensions will necessarily mis-align the points. On the other hand, if you have center aligned points, the width and height adjustments are applied around the center point so they will always be aligned regardless of what size you make them.

Food for thought.